ci/code_checks.sh

```diff
@@ -109,7 +109,13 @@ if [[ -z "$CHECK" || "$CHECK" == "lint" ]]; then
 
     # Imports - Check formatting using isort see setup.cfg for settings
     MSG='Check import format using isort ' ; echo $MSG
-    isort --recursive --check-only pandas asv_bench
+    set -o pipefail
```

Comment:
I think you should disable when it's not needed anymore. Otherwise things in other part of the script can fail, and this is not expected to be active by default.